meta.yaml

```diff
@@ -1,10 +1,13 @@
+{% set version = "2021.02.0" %}
+
 package:
   name: 'geocat-f2py'
-  version: '2021.02.0'
+  version: {{ version }}
```

Comment:
Does this pull version from the tag? it looks like it's just being set in advance of the YAML.

Reply:
No, it does not. Version is still defined in the very beginning of YAML file, and then that is used to fetch the correct tag from github repo. Only advantage here is that we are changing the worklow of creating a new release to the following:

1. Generate tag and release from the latest source code in Github
2. Use that latest tag in conda package creation. 

For reading version info from Git, addressing Anderson's comment in this [GeoCAT-comp PR](https://github.com/NCAR/geocat-comp/pull/112) would be helpful.